config/locales/en/notifiers/course/assessment/answer/comment_notifier/replied/user_notifications/email.yml

```diff
@@ -0,0 +1,15 @@
+en:
+  notifiers:
+    course:
+      assessment:
+        answer:
+          comment_notifier:
+            replied:
+              user_notifications:
+                email:
+                  subject: '%{course} New comment on %{topic}'
+                  message: >
+                    %{post_author} commented, %{post}
```

Comment:
Strangely, our `ApplicationHTMLFormattersHelper` isn't available in the notifier views, so the `format_html` method isn't available. `simple_format` actually calls the one from rails.